agnostic-svelte/src/lib/components/Pagination/Pagination.svelte

```diff
@@ -111,9 +111,10 @@
   // can be: 'start', 'center', 'end' (or empty string)
   export let justify = "";
   export let current = 1;
+  export let total = 1;
   export let pages = [];
+  export let pageGenerator = usePagination({ offset: 1 });
```

Comment:
- Not that it's likely to happen but called could pass in their own pageGenerator if they want to override correct?

Reply:
yeah if you look at the docs I updated you'll see an example of that